less/project/theme-extras.less

```diff
@@ -18,6 +18,38 @@
   .u-display-none;
 }
 
+// Header color mixin
+// Add to menu/page to enable
+// e.g. 'header-color transparent-dark'
+// Note: both arguments must be predefined variables
+// --------------------------------------------------
+.header-color-mixin(background, background-inverted);
+.header-color-mixin(transparent-light, font-color);
+.header-color-mixin(transparent-dark, font-color-inverted);
+
+.header-color-mixin(@color, @color-inverted) {
+
+.header-color.@{color} {
```

Comment:
Minor: Would it be worth moving the `@` reference out of the mixin and onto where it is called instead?
I think this would read clearer when passing `@font-color` etc.
e.g.
`.header-color-mixin(@transparent-light, @font-color);`

Reply:
Hey @cahirodoherty-learningpool, thanks for your feedback. I kept the `@` reference in the mixin for consistency with the existing `block-color` mixin. I agree that your approach would read clearer but I think it was implemented this way so that the variable name would double up as the colour class. Otherwise we would need to define a class for each colour. @guywillis was this your initial thinking?

Reply:
Okay. In that case I am happy for the consistency to be maintained 👍

Reply:
@cahirodoherty-learningpool @kirsty-hames 

That's correct - the double `@` symbol within the mixin was implemented so that a singular name could be used for both a class and a variable. Agreed that it's not the cleanest thing to read in the Less, however, the mixin will make any value passed into it a variable due to the double `@@` symbols.